### PR TITLE
Fix Auth patching for 2.9.6 ckan

### PR DESCRIPTION
**Pull request by duttonw**
https://github.com/ckan/ckan/compare/ckan-2.9.5...ckan-2.9.6\#diff-bced5766e0c0f50671e5e0f57217ef7e79f2496ed3b2c711f90daeef25dccdf1

file: ckan/lib/authenticator.py changed return method on success.

### CODE REVIEW

File: ckanext/qgov/common/authenticator.py [hunk 1]
```diff
@@ -34,18 +35,16 @@ def unlock_account(account_id):
 def intercept_authenticator():
     """ Replaces the existing authenticate function with our custom one.
     """
-    QGOVAuthenticator().orig_authenticate = UsernamePasswordAuthenticator.authenticate
-    UsernamePasswordAuthenticator.authenticate = QGOVAuthenticator().authenticate
+    # QGOVAuthenticator().orig_authenticate = UsernamePasswordAuthenticator.authenticate
+    # UsernamePasswordAuthenticator.authenticate = QGOVAuthenticator().authenticate
```
Comment (ThrawnCA) [inline ckanext/qgov/common/authenticator.py:39]: If we comment out both of these, what will cause our authenticator to actually be used?

**Comment by ThrawnCA** [comment]
Using our CKAN 2.9.5 fork works.

Using vanilla breaks because JavaScript isn't working.

**Comment by duttonw** [comment]
hmm, that still does not 'prove' it works on 2.9.6 which has the auth module change.

**Review by ThrawnCA — APPROVED** [review]
Building with both vanilla and custom is a nice touch.

**Pull request merged**